lanelet2_map_node:
  ros__parameters:
    map_path: "/home/hong/camping_cart_ws/src/lanelet2_map.osm"

    # WGS84 origin -> LocalCartesian projector 기준점
    offset_lat: 36.12491471
    offset_lon: 126.77473954
    offset_alt: 0.0

    world_frame_id: "world"
    map_frame_id: "map"
